test/spec_response.rb

```diff
@@ -451,24 +473,12 @@ def object_with_each.each
     res.body.must_be :closed?
     b.wont_equal res.body
 
-    res.body = StringIO.new
-    res.status = 205
-    _, _, b = res.finish
-    res.body.wont_be :closed?
-    b.wont_equal res.body
-
```

Comment:
Is this because of changes to `mock.rb`?

Reply:
It's testing status 205 but that's not part of the spec "calls close on #body when 204 or 304". So, what's wrong, the spec, or the name of the spec?

Reply:
Okay I restored parts of this spec, except the last line, since the body is passed through unchanged.